tests/test.js

```diff
@@ -108,3 +108,19 @@ describe('shim', function() {
 		delete Object.prototype[1];
 	});
 });
+
+describe('single function', function() {
+	var findAsFunction = function(func) {
+			return find.call(undefined, this, func, arguments[1]);
```

Comment:
we want to use `find`, but in tests `call` is used and it passes first arg as context, but we want to pass it as a just argument.